app/models/miq_server/log_management.rb

```diff
@@ -132,6 +134,26 @@ def post_logs(options)
     task.update_status("Finished", "Ok", "Log files were successfully collected")
   end
 
+  def current_log_pattern_configuration
+    get_config("vmdb").config.fetch_path(:log, :collection, :current, :pattern) || []
+  end
+
+  def current_log_patterns
+    current_log_pattern_configuration | pg_log_patterns
```

Comment:
No, this is set union.  It's a cheap way to not need a data migration.  The old configuration had the postgresql paths hardcoded:
'/opt/rh/postgresql92/root/var/lib/pgsql/data/*.conf'
'/opt/rh/postgresql92/root/var/lib/pgsql/data/pg_log/*'

This new code reads the existing current pattern array and adds the postgres log patterns from environment variables, and does set union to ensure we only add unique elements in case the current configuration still has the old hardcoded values.  It's not a big deal with they migrate and never remove these values as we won't find logs in old locations so it's benign to have them there.